clang/www/analyzer/index.html

```diff
@@ -96,11 +96,11 @@ <h2>Download</h2>
    <h3 style="margin:0px;padding:0px">Mac OS X</h3>
    <ul>
     <li>Latest build (10.8+):<br>
-     <!--#include virtual="latest_checker.html.incl"-->
+     <b><a href="downloads/checker-279.tar.bz2">checker-279.tar.bz2</a></b> (built November 14, 2016)
```

Comment:
The entire downloads section should probably be nuked. The macOS section is completely out of date and meaningless. The rest boils down to "just download clang how you normally download clang".

Instead, we can point them to the scan-build and CodeChecker page, or to their IDE's documentation.

I suggest:
```txt
The static analyzer is part of Clang. Please refer to the
<a href="https://releases.llvm.org/download.html">LLVM
releases page</a> for further instructions.

The official LLVM releases also include <a href="scan-build.html">scan-build</a>.
If you're installing Clang from a different source, such as
a linux package repository, then scan-build may be packaged
separately as an individual package, or as part of
a "clang tools" package.

If your IDE is using Clang, it may natively integrate the static analyzer.
On macOS, the easiest way to use the static analyzer is to invoke it
<a href="https://clang.llvm.org/docs/analyzer/user-docs/UsingWithXcode.html">directly
from Xcode</a>.

Additionally, if you're using <a href="https://clang.llvm.org/extra/clang-tidy/">clang-tidy</a>,
you can naturally make the static analyzer run alongside clang-tidy
by enabling the <a href="https://clang.llvm.org/extra/clang-tidy/checks/list.html">clang-analyzer</a>
checks.
```

(Could be a follow-up commit.)

Reply:
@haoNoQ Thanks for the insights!
I'll also create another PR to clean up the index.html and finish the developer section.
There will be questions about what to do with open projects and checker ideas, but let's discuss it later in that PR.